package.json

```diff
@@ -17,9 +17,11 @@
         "serverid": "showdown",
         "nickname": "",
         "password": "",
-        "room": ""
+        "room": "",
```

Comment:
I think these should really be generalized - ie/ we should be able to configure which rooms get which messages from which repositories (as opposed to hardcoding it in the code as is done currently).

@xfix because this is something that lends itself to a config structure which supports nesting (for example, an arbitrary number of showdown or discord servers and their respective rooms/channels as sinks, which just becomes unwieldy with environment variables). some of these values like `secret` lend themselves to being environment variables, but specifiying multiple becomes awkward (though perhaps JSON values could help here)?

Reply:
I've updated this to take JSON for the channels for Discord notifications